README.md

```diff
@@ -14,8 +14,8 @@ Under the hood, think of Streamwall as a specialized web browser for mosaicing v
 
 ## Setup
 
-1. Install dependencies: `npm install`
-2. Launch: `npm start`
+`git clone https://github.com/chromakode/streamwall.git && cd streamwall && npm install && npm start`
```

Comment:
I would leave the separate instructions for cloning (add that as step #1), installing, and starting.  Most people cloning to make modifications would probably use a process to watch for changes like nodemon or pm2.  While I do understand that all three commands are in the same line and could be copied only up to that point.. not all users will know.  Suggest keeping the line you added, add a step 1 for clone, and label your line as "All-in-one clone & install" or something of the like.

Reply:
Sounds good, will make the change when I get home. Shall I also use npm start-local instead of npm start as suggested on the issue I opened?